include/libsemigroups/hpcombi.hpp

```diff
@@ -1072,17 +1294,27 @@ namespace libsemigroups {
                         std::vector<uint8_t>(ran.begin(), ran.end()),
                         M);
   }
+#else
+  template <typename Int>
+  [[nodiscard]] HPCombi::PPerm16
+  make<HPCombi::PPerm16>(std::initializer_list<Int> const& dom,
+                         std::initializer_list<Int> const& ran,
+                         size_t                            deg = 16);
```

Comment:
I assume that we add the `#ifndef LIBSEMIGROUPS_PARSED_BY_DOXYGEN` to get around the unsightly `enable_if_is_same_stuff`. If this is the only reason, then the arguments of this doxygen-only declaration should be the same as those in the actual declaration.

It's not clear to me which of `size_t deg = 16` or `size_t M` is correct.